app/services/jbpm_process_service.rb

```diff
@@ -26,14 +26,17 @@ def template
   end
 
   def process_options
-    groups =
-      ContextService.new(request.context).as_org_admin do
-        request.workflow.group_refs.map { |ref| Group.find(ref) }
-      end
-
-    { 'request' => request.as_json,
-      'groups'  => groups.as_json,
-      'stages'  => request.stages.as_json }
+    options = nil
+    ContextService.new(request.context).as_org_admin do
+      groups = request.workflow.group_refs.map { |ref| Group.find(ref) }
+
+      options = {
+        'request' => request.as_json,
+        'groups'  => groups.as_json,
+        'stages'  => request.stages.as_json
```

Comment:
Note `request.stages.as_json` needs to access rbac, therefore the whole method should be placed under the scope of `as_org_admin`. It is the reason this PR is created.